src/wp-includes/block-bindings/sources/pattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * The "pattern" source for the Block Bindings API. This source is used by the
+ * Partially Synced Patterns.
```

Comment:
Nit: It's now called "Synced Pattern Overrides" or just "Pattern Overrides". 😅

Reply:
Yes, we have to incorporate the changes made in this pull request as well: [link](https://github.com/WordPress/gutenberg/pull/58434).